### PR TITLE
remove coffee from deps

### PR DESCRIPTION
**Pull request by imgarylai**
Appreciate @voleoo fixed the tests! 

This PR is relevant to #138 #139. Let me create smaller PRs for review. 

The coffee file was written in js since Coffee Script was dropped around Spree 4.0. We could remove coffee from dependencies.

### CODE REVIEW

File: app/overrides/add_documents_to_main_navigation.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Deface::Override.new(
-  virtual_path:  'spree/layouts/admin',
-  insert_bottom: '#main-sidebar',
+  virtual_path:  'spree/admin/shared/_main_menu',
+  insert_bottom: 'nav',
   partial:       'spree/admin/shared/menu/documents_tab',
   name:          'documents_tab'
 )
```
Comment (voleoo) [inline app/overrides/add_documents_to_main_navigation.rb:6]: Could you remove this changes from PR.

We should have it in master but with another PR.

Also let's keep support  Spree < 4
```
if Spree.version.to_f < 4.0
  Deface::Override.new(
    virtual_path:  'spree/layouts/admin',
    insert_bottom: '#main-sidebar',
    partial:       'spree/admin/shared/menu/documents_tab',
    name:        'documents_tab'
  )
else
  Deface::Override.new(
    virtual_path:  'spree/admin/shared/_main_menu',
    insert_bottom: 'nav',
    partial:       'spree/admin/shared/menu/documents_tab',
    name:          'documents_tab'
  )
end
```

Comment (imgarylai) [reply app/overrides/add_documents_to_main_navigation.rb:6]: no problem. Let me create another PR.

Comment (voleoo) [reply app/overrides/add_documents_to_main_navigation.rb:6]: > no problem. Let me create another PR.

Let's add to PR #143

Comment (imgarylai) [reply app/overrides/add_documents_to_main_navigation.rb:6]: Added to #143

Comment (voleoo) [reply app/overrides/add_documents_to_main_navigation.rb:6]: Could you just revert file with `git checkout master app/overrides/add_documents_to_main_navigation.rb`

Comment (imgarylai) [reply app/overrides/add_documents_to_main_navigation.rb]: updated.

**Comment by voleoo** [comment]
Hey, @damianlegawiec I think it can be merged to master

**Comment by damianlegawiec** [comment]
Thanlk you @imgarylai @voleoo 👍

**Pull request merged**